envsec/pkg/envsec/init.go

```diff
@@ -20,7 +20,7 @@ var (
 )
 
 const (
-	dirName       = ".jetpack.io"
+	dirName       = ".jetify.com"
```

Comment:
This is gonna cause projects to be reinitialized, but that's OK.

Reply:
Should we do `.jetify`?